site/app/templates/submission/homework/CurrentVersionResults.twig

```diff
@@ -8,66 +8,103 @@
                         <i class="fas fa-download" title="Download the file"></i></button>
                 {% endif %}
-
-                {% if loop.index != files|length %}
-                    <br />
-                {% endif %}
+                 </div>
             {% endfor %}
             {% if can_download and files|length > 1 %}
                 <br />
-                <br />
-                Download all files: <a onclick='downloadSubmissionZip("{{ gradeable_id }}", "{{ user_id }}", "{{ display_version }}", "submission")' aria-label="Download zip of all files" class="key_to_click" tabindex="0"><i class="fas fa-download" title="Download zip of all files"></i></a>
+                <div class="flex-row">
+                    <span>Download all files:</span>
+                    <a aria-label="Download zip of all files"
+                       onclick='downloadSubmissionZip("{{ gradeable_id }}", "{{ user_id }}", "{{ display_version }}", "submission")'
+                       class="key_to_click"
+                       tabindex="0"
+                    >
+                        <i class="fas fa-download" title="Download zip of all files"></i>
+                    </a>
+                </div>
             {% endif %}
         </div>
-        <div class="box col-md-6">
+        <div class="box col-md-6 submission-details">
             {% if first_access_time is not null and first_access_time is not empty %}
-                first access timestamp: {{ first_access_time|date('m/d/Y h:i:s A T') }}<br />
+                <span class="flex-row">
+                    <span>First access timestamp:</span>
+                    <span>{{ first_access_time|date('m/d/Y h:i:s A T') }}</span>
+                </span>
             {% endif %}
-            <span id="submission_timestamp"> submission timestamp: {{ submission_time|date('m/d/Y h:i:s A T') }}</span> <br />
-            days late: {{ days_late }} (before extensions)<br />
+            <span class="flex-row">
+                <span id="submission_timestamp">Submission timestamp:</span>
+                <span> {{ submission_time|date('m/d/Y h:i:s A T') }}</span>
+            </span>
+            <span class="flex-row">
+                <span id="submission_timestamp">Days late: </span>
+                <span>{{ days_late }} (before extensions)</span>
+            </span>
             {% if results is defined %}
-                grading time: {{ grade_time }} seconds<br />
+                <span class="flex-row">
+                    <span id="submission_timestamp">Grading time:</span>
+                    <span>{{ grade_time }} seconds</span>
+                </span>
                 {% if num_autogrades == 1 %}
-                    queue wait time: {{ wait_time }} seconds<br />
+                    <span class="flex-row">
+                        <span id="submission_timestamp">Queue wait time:</span>
```

Comment:
There are a bunch of element all with the same id `submission_timestamp`
You might want to give them all a unique id, or switch `submission_timestamp` to be a class if it is used for css

Reply:
added, thanks!